src/lib/IdlenessDetector.ts

```diff
@@ -79,26 +83,21 @@ export class IdlenessDetector {
    * onNewActivityDetected is executed only if the idleness job is not running
    * and result can cancel the restart of the job
    */
-  private registerUserActivity(
-    inactiveDurationInMillis: number,
-    idlenessDetectionCheckThreshold: number,
-  ): void {
+  private registerUserActivity(): void {
     if (!this.idlenessDetectionJob) {
       this.onNewActivityDetected?.();
     }
     this.lastActivityTimestampInMillis = Date.now();
     if (this.idlenessDetectionJob === undefined) {
-      this.startIdlenessDetection(inactiveDurationInMillis, idlenessDetectionCheckThreshold);
+      this.startIdlenessDetection();
     }
   }
 
-  private startIdlenessDetection(
-    inactiveDurationInMillis: number, idlenessDetectionCheckThreshold: number,
-  ): void {
+  private startIdlenessDetection(): void {
     this.idlenessDetectionJob = this.scheduler.schedule(
       'Idleness detector',
-      () => this.verifyUserIdleness(inactiveDurationInMillis),
-      idlenessDetectionCheckThreshold,
+      () => this.verifyUserIdleness(this.inactiveDurationInMilliseconds),
```

Comment:
Du coup même ici, on pourrait supprimer le paramètre non ?

Reply:
je comprends pas, on utilise ici la variable de la classe pour initialiser le scheduler, j'en ai forcément besoin non ?

Reply:
``En fait la méthode `verifyUserIdleness` pourrait se sourcer directement sur `this.inactiveDurationInMilliseconds` :
```typescript
  private verifyUserIdleness(): void {
    if (this.idleTimeInMillis() > this.inactiveDurationInMilliseconds) {
      this.onIdlenessDetected?.();
      this.stopIdlenessDetection();
    }
  }
```